addons/account/static/src/css/account.css

```
@@ -72,3 +72,7 @@
         margin-left: 2px;
     }
 }
+
+.o_account_move_form_view .o_cell:has(div[name="journal_div"]:empty) {
```

Maybe we should only select `.o_cell` where the `journal_div` is an immediate child element to be a bit more precies?
```suggestion
.o_account_move_form_view .o_cell:has(>div[name="journal_div"]:empty) {
```
It would probably not make a difference in practice but I don't know the details of the form view / `.o_cell` generation either.